### PR TITLE
improve integer number validation message

### PR DESCRIPTION
**Pull request by ogmzh**

### CODE REVIEW

File: typescript-react/src/components/validation/type.ts [hunk 1]
```diff
@@ -12,7 +13,7 @@ import { validatorCreatorFactory } from './validatorCreatorFactory';
 export const isIntegerCreator =
   validatorCreatorFactory<void, Numeric, any, any>({
     getValidatorErrorMessage: (_, input) => !(Number(input) <= INT_MAX_VALUE && Number(input) >= INT_MIN_VALUE) ?
-      'Must be a whole number with up to 10 digits' :
+      `Must be a whole number between ${formatNumberToDecimals(INT_MIN_VALUE, 0)} and ${formatNumberToDecimals(INT_MAX_VALUE, 0)}` :
```
Comment (zapov) [inline typescript-react/src/components/validation/type.ts:16]: Shouldn't this be some other constant instead of being calculated here every time?

Comment (ogmzh) [reply typescript-react/src/components/validation/type.ts]: Sure, changed

Comment (zapov) [reply typescript-react/src/components/validation/type.ts]: I meant the whole message :)

Comment (zapov) [reply typescript-react/src/components/validation/type.ts]: I mean, its most likely not important :D
It just stands out to me when I read it

Comment (ogmzh) [reply typescript-react/src/components/validation/type.ts]: Maybe this time you'll be happy :D

Comment (ogmzh) [reply typescript-react/src/components/validation/type.ts]: Also, squashed the commits

Comment (zapov) [reply typescript-react/src/components/validation/type.ts]: :)
When I look at that code the behavior I prefer is to have that message string as constant created only once and only when needed
This code does create it only once, but I dont know if it created only when needed.
I mean its probably not worth it, but when you ignore such reasoning all the time you end up bleeding from 1000 cuts

**Pull request merged**